packages/client/vite.config.ts

```diff
@@ -220,12 +216,12 @@ const copyPlugin = (
               transform,
             },
             {
-              src: "public-workspace/images/icons/*",
-              dest: "images/icons",
+              src: "public-workspace/images/*",
```

Comment:
not sure why this was arranged like this .. probably another gaffe of mine from the client/workspace convergence .. should all be ship-shape with this mod